source/_integrations/elkm1.markdown

```diff
@@ -119,7 +122,15 @@ The complete list of trouble statuses are:
 
 To add `ElkM1` to your installation, go to **Configuration** >> **Integrations** in the UI, click the button with `+` sign and from the list of integrations select **Elk-M1 Control**.
 
-Alternatively, add the following section to your `configuration.yaml` file:
+Alternatively, configuration through the `configuration.yaml` file
+is supported (example below).
+
+Both methods of configuration support "auto configuration". This works by
+reporting only elements on the ElkM1 that have a "Name" configured.
+So, for example, if counter #11 on the panel was configured with
+the name "Test counter" then this element would show up in Home Assistant. If
+an element is being used but does not have a name configured then
+it will not appear in Home Assistant through the auto configuration feature.
```

Comment:
```suggestion
Alternatively, configuration through the `configuration.yaml` file
is supported (example below).

Both methods of configuration support "auto configuration". This works by
reporting only elements on the ElkM1 that have a "Name" configured.
So, for example, if counter #11 on the panel was configured with
the name "Test counter" then this element would show up in Home Assistant. If
an element is being used but does not have a name configured then
it will not appear in Home Assistant through the auto-configuration feature.
```